surveys/2022-annual-survey/questions.md

```diff
@@ -26,40 +26,62 @@ Type: select one
 
 ## For previous Rust users
 
-### As you have indicated that you're no longer using Rust, what prompted you to participate in this survey?
+### Why did you stop using Rust?
 
-Type: select all that apply (optional)
+Type: select all that apply
 
-- I plan to return to using Rust in the future
-- I consider myself part of the Rust community
-- Specifically to provide feedback on why I stopped using Rust
-- To provide feedback on Rust in general
-- Curiosity
-- Other (open response)
+- No benefit or not appropriate for my projects
```

Comment:
I'm not sure if this category covers folks who simply don't have the time to use it for their projects. For example, someone who is working on web app front ends maybe *could* use Rust, but it's not really super noteworthy that they are not. I wonder if it's the fact that this choice makes it seem much more negative than it might be. 

Perhaps we could split this into the following:
* I no longer have the opportunity to use Rust due to external factors
* Rust did not help me achieve my goals